metro.config.js

```diff
@@ -5,13 +5,29 @@
  * @format
  */
 
-module.exports = {
-  transformer: {
-    getTransformOptions: async () => ({
-      transform: {
-        experimentalImportSupport: false,
-        inlineRequires: true,
+const path = require('path');
+const { getDefaultConfig } = require("metro-config")
+
+module.exports = (async () => {
+  const {
+    resolver: { sourceExts, assetExts }
+  } = await getDefaultConfig()
+  return {
+    transformer: {
+      getTransformOptions: async () => ({
+        transform: {
+          experimentalImportSupport: false,
+          inlineRequires: true
+        }
+      }),
+      babelTransformerPath: require.resolve("react-native-svg-transformer")
+    },
+    resolver: {
+      extraNodeModules: {
+        '@i18n': `${__dirname}/node_modules/@lifeomic/react-native-sdk/lib/i18n`,
       },
-    }),
-  },
-};
+      assetExts: assetExts.filter(ext => ext !== "svg"),
+      sourceExts: [...sourceExts, "svg"]
+    },
+  }
+})();
```

Comment:
It's a bummer this needs to be replicated in consuming repos, right?  But maybe inevitable?  Maybe the best we could do long-term is provide helper/utils and/or a metro.config to copy.  Whether as part of current task or creating a new task, can you make sure we somehow document this in the SDK?  It could be a link to this repo's metro.config.js (or the example app's in the SDK), but I just want to make sure if someone does `init`s a vanilla react-native app, then adds the SDK, we've roughly got all the steps documented to make it run locally.

Reply:
I think we should write a small script that runs on post install and automatically updates the consuming project's config. At minimum I will document it though.

Reply:
That makes for a nice experience, but also a dangerous one if the consuming lib is already established and has its own metro.config already configured to its liking.